transformations/yoda_transform/transformation.py

```diff
@@ -0,0 +1,130 @@
+import itertools
+
+import spacy
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+"""
+Base Class for implementing the different input transformations a generation should be robust against.
+"""
+"""
+Yoda Speak implementation borrowed from here: https://github.com/yevbar/Yoda-Script, with minor mofifications to fix
+deprecation and to change some stylistic features
+"""
+
+punctuation = [",", ".", ";", "?", "!"]
+nlp = spacy.load("en_core_web_sm")
```

Comment:
You might want to use spacy like [this](https://github.com/GEM-benchmark/NL-Augmenter/blob/d39f0e5f57a79937296351996617ac03ee80fb74/transformations/close_homophones_swap/transformation.py#L55).